src/main/scala/org/apache/spark/shuffle/ucx/perf/UcxPerfBenchmark.scala

```diff
@@ -123,7 +139,8 @@ object UcxPerfBenchmark extends App with Logging {
             if (requestInFlight.decrementAndGet() == 0) {
               printf(s"Received ${options.numOutstanding} block of size: ${stats.recvSize}  " +
                 s"in ${stats.getElapsedTimeNs / 1000} usec. Bandwidth: %.2f Mb/s \n",
-                (options.blockSize * options.numOutstanding) / (1024.0 * 1024.0 * (stats.getElapsedTimeNs / 1e9)))
+                (options.blockSize * options.numOutstanding * options.numThreads) /
```

Comment:
it seems to the log is still printed for `options.numOutstanding` requests. am i missing anything?